README.md

```diff
@@ -56,7 +56,7 @@ NOTIFICATION_STORE_PROVIDER = {
 
 # This setting already exists in the LMS, please
```

Comment:
By default django looks for migrations in `migrations` package for each app but we can override this on per-apps basis by updating `MIGRATION_MODULES` setting so that edx-platform can find migrations for `edx_notifications` app, whose migrations package is not at root of the app.